src/node.ts

```diff
@@ -13,6 +13,7 @@ import {
 } from './utils'
 
 const pathSeparator = '/'
+const pathSeparatorByte = 47
```

Comment:
personal preferenece: could we have constants be all caps?

Reply:
not a bad idea